registrationform.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Registration Form</title>
+</head>
+
+<body>
+
+    <!-- Full Name Email Passord inputs. -->
+
+    <fieldset>
+        <legend>Basic User Information:</legend>
+
+        <label for="full-name"> Full name:</label>
+        <input type="text" id="full-name" name="full-name" value=""> <br>
+        <label for="email"> Email:</label>
+        <input type="text" id="email" name="email" value=""> <br>
+        <label for="password"> Password:</label>
+        <input type="password" id="password" name="password" value="">
+        <label for="password"> Confirm Password:</label>
+        <input type="password" id="confirm-password" name="confirm-password" value=""> <br>
+
+        <!-- Gender Input. -->
+        <p>Gender:
```

Comment:
Inside `<p>` should be text. Not form elements

Reply:
You can put "Gender" into a <label> tag with no "for" attribute. It should work

Reply:
Ok. I found the problem I think. Thank you !